core/cat/mad_hatter/plugin.py

```diff
@@ -45,29 +45,26 @@ def __init__(self, plugin_path: str, active: bool):
         #   but they are created and stored in each plugin instance
         self._hooks = [] 
         self._tools = []
-
         self._active = False
```

Comment:
@pieroit  I leaved the active attribute, although at the moment it is not used i think it is useful to quickly check if a plugin have loaded hook and tools